scania-digital-design-system/sdds
# fix(button): correct disabled secondary, ghost button icon color

Before 🏚️:
<img width="144" alt="Screenshot 2023-03-21 at 16 13 13" src="https://user-images.githubusercontent.com/8556022/226651558-946499ce-d1e2-4cb3-8d59-e283690e619c.png">

After 🏡✨:
<img width="141" alt="Screenshot 2023-03-21 at 16 13 36" src="https://user-images.githubusercontent.com/8556022/226651593-90913491-73be-4a61-b8a1-a964eb39dc48.png">

**Solving issue**  
Fixes: #DTS-1256

**How to test**  
1. Go to deploy preview
2. Check in button component
3. Run make the button disabled and give it an icon
4. Test secondary and ghost type buttons
